context/app/static/js/components/Detail/provenance/NodeElement/NodeElement.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import useProvenanceStore from 'js/stores/useProvenanceStore';
+import { Asterisk } from './style';
+
+const provenanceStoreSelector = (state) => state.uuid;
+
+function NodeElement(props) {
+  const { node, title, columnWidth } = props;
+  const style = node.nodeType === 'input' || node.nodeType === 'output' ? { width: columnWidth || 100 } : null;
+
+  const displayTitle = title || node.title || node.name;
+
+  const uuid = useProvenanceStore(provenanceStoreSelector);
+
+  function tooltip() {
+    let output = '';
+
+    // Node Type
+    if (node.nodeType === 'step') {
+      output += `<small>Step ${(node.column - 1) / 2 + 1}</small>`;
+    } else {
+      let { nodeType } = node;
+      nodeType = nodeType.charAt(0).toUpperCase() + nodeType.slice(1);
+      output += `<small>${nodeType}</small>`;
+    }
+
+    // Title
+    output += `<h5 class="text-600 tooltip-title">${node.title || node.name}</h5>`;
+
+    // Description
+    if (typeof node.description === 'string' || (node.meta && typeof node.meta.description === 'string')) {
+      output += `<div>${node.description || node.meta.description}</div>`;
+    }
+
+    return output;
+  }
```

Comment:
How is this supposed to work? I don't see a tooltip when I hover.  I also didn't see this mentioned in the PR so maybe this is just for debugging?

Reply:
Oh sorry, should have mentioned. Most of this is copy over from the default node included in `react-workflow-viz` so not my code, I'll create a new tooltip in a follow-up PR, but wanted to make minimal changes here.

Reply:
That makes sense @john-conroy and seems like a good way forward.